epam-promo/components/inputs/docs/textInput.doc.ts

```diff
@@ -1,8 +1,9 @@
 import { DocBuilder } from '@epam/uui-docs';
 import { TextInputProps } from '@epam/uui-components';
-import { TextInput, TextInputMods } from '../TextInput';
+import { TextInput } from '../TextInput';
 import { iEditable, iHasPlaceholder, onClickDoc, isDisabledDoc, isReadonlyDoc, isInvalidDoc, iconDoc, iconOptionsDoc, dropdownTogglerDoc } from '../../../docs';
 import { DefaultContext, FormContext, TableContext, IHasEditModeDoc } from '../../../docs';
+import { TextInputMods } from "@epam/uui";
```

Comment:
use here TextInputProps interface which union TextInputMods and core text input props

Reply:
done